### PR TITLE
MAID-2410 feat/Expose functions to verify current network connection state

### PR DESCRIPTION
**Pull request by bochaco**
Exposing some functions to verify current state of network connection. Also, replace all `functions` for arrow functions.

### CODE REVIEW

File: examples/standalone/scripts/idata_asymmetric.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ let run = async () => {
 
 run()
 
-function wait () {
```
Comment (joshuef) [inline examples/standalone/scripts/idata_asymmetric.js]: Is it possible to put the `wait` function in a helper/util file if we're using it all over? (can we pass in `EXIT_CONDITION`)?

File: test/index.js [hunk 2]
```diff
@@ -17,6 +17,23 @@ describe('Smoke testing', () => {
   it('confirms the CONSTANTS are exposed', () => {
     should(CONSTANTS).be.Object();
   });
+
+  it('confirms the full list of contants', () => {
```
Comment (joshuef) [inline test/index.js]: `contants` => `con<s>tants`

File: examples/standalone/scripts/idata_asymmetric.js (hunk 1 above)
Comment (bochaco) [reply examples/standalone/scripts/idata_asymmetric.js]: Yes we can do that, let me send a commit with such an utility function. The only comment is that if you grab one of these files to run them isolated you'll need to make sure to also include/copy that dependency file.

**Comment by bochaco** [comment]
@joshuef , I pushed a commit based on your review. I'm using `wait` package from npm which provides the utility you suggested.

**Pull request merged**